src/proctored-exam-settings/proctoredExamSettings.scss

```diff
@@ -3,3 +3,10 @@ legend {
     font-size: 1.0rem;
     font-weight: 700;
 }
+
+.form-check:focus-within {
```

Comment:
NIT: What do you think of using the same styling as the other components on the page? I just copied these styles from the developer tools.

```
  border: 1px solid #6582dd;
  border-radius: 5px;
  box-shadow: 0 0 0 1px #6c83cd;
```